spec/controllers/session_controller_spec.rb

```diff
@@ -659,6 +659,22 @@ def post_login
         end
       end
     end
+
+    context 'rate limited' do
+      it 'rate limits login' do
+        SiteSetting.max_logins_per_ip_per_hour = 2
+        RateLimiter.stubs(:disabled?).returns(false)
```

Comment:
Okay, it's been used in many other places. I'll look into it and push a new commit for it.